### PR TITLE
[WIP] Update Community translation files

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Update `Community` content bucket translation files

### CODE REVIEW

File: src/content/translations/zh/community/get-involved/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- 观看或参与[核心开发人员电话会议](https://www.youtube.com/playlist?list=PLaM7G4Llrb7zfMXCZVEXEABT8OSnd4-7w)
+- [生态系统支持计划的愿望清单](https://esp.ethereum.foundation/wishlist/) - 适用于工具、文档和基础设施领域，以太坊生态系统支持计划正在积极寻求这些领域的资助申请
+- [Web3Bridge](https://www.web3bridge.com/) - 加入有抱负的 web3 社区，一起积极寻找、培训和支持整个非洲的数百名开发人员和社区成员
+
+## 研究人员 & 学者 <Emoji text=":mag:" size={1} />‍ {#researchers-and-academics}
+
+您是否有数学、加密或经济学方面的背景？ 您可能会对以太坊生态系统中正在进行的一些前沿工作感兴趣
+
+- [Challenges.ethereum.org](https://challenges.ethereum.org/) - 一系列高额研究奖金，您可以从中赚取 >100,000 美元
+- <a href=”https://ethresear.ch“>ethrear.ch</a> - 以太坊的主要研究论坛，也是世界上最具影响力的加密经济学论坛
```
Comment (minimalsm) [inline src/content/translations/zh/community/get-involved/index.md:27]:
```suggestion
- [ethrear.ch](https://ethresear.ch) - 以太坊的主要研究论坛，也是世界上最具影响力的加密经济学论坛
```

**Comment by minimalsm** [comment]
Failed after 90 minutes. Re-running

**Review by minimalsm — COMMENTED** [review]
@corwintines not really sure why this is still failing on Gatsby's end 🤔

**Comment by corwintines** [comment]
Going to break this out

**Pull request closed without merging**